projects/pulsar_adc_pmdz/common/pulsar_adc_pmdz_pmod.txt

```diff
@@ -0,0 +1,9 @@
+Pin            Port               Schematic_name     System_top_name        IOSTANDARD  Termination
+
+# pulsar_adc_pmdz
+
+PMOD J5 [8]    PMOD JA [4]        SCK                pulsar_adc_spi_sclk    LVCMOS33    #N/A
```

Comment:
I think that we should update the Port column with the name specified in the carrier's schematic: JA1_P/N and the pin column with the pin number.